kale/task.py

```diff
@@ -195,6 +198,9 @@ def run(self, *args, **kwargs):
         """
 
         self._setup_task_environment()
+        if not self.should_run_task(*args, **kwargs):
+            self.republish()
```

Comment:
This call to `republish` is missing arguments.

Reply:
Those lines should not even be there. Mistakenly committed.